6_3_C.py

```diff
@@ -0,0 +1,31 @@
+def get_letters():
+    """
+    :return: List that includes all the letters
+    """
+    small_letter = [chr(i) for i in range(ord('a'), ord('z') + 1)]
+    capital_letter = [chr(i) for i in range(ord('A'), ord('Z') + 1)]
+    return small_letter + capital_letter + [" "]
+
+
+def count_words(text):
+    """
+    :param text: String that includes words
+    :return: A dictionary of the lengths of the words in the text without punctuation
+    """
+    only_letter = "".join([letter for letter in text if letter in get_letters()])

```

Comment:
I like the use of the previous function but in this case it is not the most efficient sollution. How can this be done more efficiently?